### ISSUE TITLE
Copyedits for Principals document

### ISSUE THREAD

**Issue by madeleinerothberg** [w3c/publ-a11y#619]
Madeleine's copyedit suggestions, mostly for Section 2 but one for Section 3

An overall issue: Section 3 is called “Key Accessibility Information” and lists all the accessibility metadata. However, the start of section 2 implies that the 3 recommended topics are the key information. Later in section 2 it says that all the info is key. We should either rename section 3 or change the use of the word “key” in section 2. The smallest change would be to re-word the start of section 2.1.2 with a different word to emphasize the importance of the 3 recommended topics. It leaves open the notion that there is some “non-key” metadata we have chosen not to include which I’m not sure there is since the final section on Additional accessibility information covers the outliers. But that is the simplest solution unless others think more changes are needed.

Changes suggested below assume we are not renaming section 3 but need to better name the 3 recommended topics. There are 8 suggestions each with a number and then the number of the document section follows.

2.1.2 Intro sentence currently reads:
“When focusing on the accessibility of any digital publication, several areas of key information come to mind:”
I suggest:
“There are several pieces of information that are central to describing the accessibility of a digital publication:” (central could also be important, critical, most important, etc.)

2.1.2 First bullet, last sentence. Does “low vision and dyslexia” suggest that people have both? Would “or” be better?

2.1.2 Second bullet – after the first sentence, should there be something about the text itself, before discussing image description? It could read something like this, with some other edits also made for smoothness, including lower-case for reading systems:
“People who use a screen reader need to know if all the content in the title will be accessible to them. The text of the publication must be available for conversion to speech or braille. In addition, when images have text descriptions (alt text), indicating that in metadata will let users know that they will be not missing out on essential information. Blind users will greatly benefit from this information as will individuals who use the read aloud feature in reading systems.”

2.1.2 Last paragraph before the Note currently reads:
“The other areas provide details about specific features or shortcomings in publications. It is expected that these other areas of key information will give people what they need to make an informed choice to read a particular e-book.”

I suggest:
“The other metadata topics discussed below provide further details about specific features or shortcomings in a publication to give people the information they need to make an informed choice when selecting a particular digital publication.”

2.1.3
Currently reads:
“In this document are examples of important metadata that is expected to be in a wide range of publications. However, we do not have examples of all possible features that the metadata can express. The techniques have more values than are shown in the examples.”

I suggest:
“This document showcases examples of important metadata that is expected to be present in a wide range of publications. Available metadata schemas can support statements about many other features. The techniques documents provide additional examples.”

2.1.5
Should the links to techniques documents include the full name of each document instead of just the standard’s name?

2.2
Currently reads:
“Although the display statements listed in this document and in the techniques are recommended for use, it is possible to use alternative statements if the situation requires.”

I suggest the following, which adds a note about localization to encourage implementors to use the provided strings. Remove if that is too opinionated:
“Although the display statements listed in this document and in the techniques documents are recommended for use, and have been translated to support localization, implementors may wish to use different wording in some situations.”

3.1.3
The Note is missing a period at the end of the last sentence. If the reason to skip it is that the preceding text is the recommended display text, which doesn’t have a period, the recommended text can be in quotes as it is in some other notes and then the period can be outside the quotes.

**Comment by mattgarrish** [comment]
> An overall issue: Section 3 is called “Key Accessibility Information” and lists all the accessibility metadata.

Agree. I raised this previously, too. The non-key information I assume is everything that falls under the additional information section, but having two recommended sections seems to narrow it further.

I wonder if we should do the reverse: drop the word "key" everywhere except for that lead-in that states what the two priority fields are?

If everything is key, then it stops making sense to call it that.

**Comment by GeorgeKerscher** [comment]
Hi,
 
In the intro there is an explanation of the three key pieces. Conformance is in there. I agree we need to improve this.
 
Who will do this, Matt or me?
 
I know Matt is work on IDs in Examples. I suppose after that is merged?
 
Best
George

**Comment by GeorgeKerscher** [comment]
I implemented these suggestions in PR 625

**Comment by GeorgeKerscher** [comment]
All copy edits made and approved.
